Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -208,8 +208,10 @@ void CVolumeGC::LoadBannerFile() const
 
   GCBanner banner_file;
   std::unique_ptr<IFileSystem> file_system(CreateFileSystem(this, PARTITION_NONE));
-  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
+  if (!file_system)
+    return;
 
+  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
```

Comment:
@BhaaLseN File sizes are u32s originally in the FST, so it makes no difference in practice. PR #2820 doesn't convert it to u64 inside GCWiiFileSystem, but master does. I'll clean up the casting for now.